src/utility/hexagons.js

```diff
@@ -823,11 +834,16 @@ var HexGrid = class HexGrid {
 		this.showCreaturehexes();
 	}
 
+    cleanHex (hex) {
+   	hex.cleanDisplayVisualState()
```

Comment:
Any punctuation required after this?

Reply:
ES6 does not require punctuation, but in order for it to be backwards compatible with older browsers without transpiling it should have one. So I will add it in my next few changes.

Reply:
To clarify, Javascript inherently supports no trailing semi-colons through ASI (Automatic Semi-colon Insertion) and this has nothing to do with old browsers (it is part of the original code base).

However, always using semi-colons is a good thing, because if files are concatenated, you never know the state of other libs that might get concatenated with your code. This is why in the mid 2000's and sometimes still today you can find open source libs that prefix all JS files with a leading semi-colon for no apparent reason... It's to fix the lack of semi-colon when concatenating multiple JS sources. Trailing semi-colons are a good practice and should not be neglected ;)